gettsim/transfers/arbeitsl_geld.py

```diff
@@ -42,71 +45,147 @@ def arbeitsl_geld_m(
     return out
 
 
-def arbeitsl_monate_gesamt(
-    arbeitsl_monate_lfdj: int,
-    arbeitsl_monate_vorj: int,
-    arbeitsl_monate_v2j: int,
+def restliche_anspruchsdauer(
```

Comment:
```suggestion
def arbeitl_geld_restliche_anspruchsdauer(
```
For functions we should typically add the name of the transfer.